src/Features/Lsif/Generator/CompilerInvocation.cs

```diff
@@ -34,6 +34,7 @@ public CompilerInvocation(Compilation compilation, HostLanguageServices language
         public static async Task<CompilerInvocation> CreateFromJsonAsync(string jsonContents)
         {
             var invocationInfo = JsonConvert.DeserializeObject<CompilerInvocationInfo>(jsonContents);
+            Assumes.Present(invocationInfo);
```

Comment:
It's a bit weird for me to see Assumes.Present (which is normally used in VS services) being used here. I'm kinda surprised Microsoft.VisualStudio.Validation is actually referenced by this project at all.